src/Assets/Api.php

```diff
@@ -121,29 +121,6 @@ function() use ( $handle ) {
 		}
 	}
 
-	/**
-	 * Queues a block script in the frontend.
-	 *
-	 * @since 2.5.0
-	 * @since 2.6.0 Changed $name to $script_name and added $handle argument.
-	 * @since 2.9.0 Made it so scripts are not loaded in admin pages.
-	 * @deprecated 4.5.0 Block types register the scripts themselves.
-	 *
-	 * @param string $script_name  Name of the script used to identify the file inside build folder.
-	 * @param string $handle       Optional. Provided if the handle should be different than the script name. `wc-` prefix automatically added.
-	 * @param array  $dependencies Optional. An array of registered script handles this script depends on. Default empty array.
-	 */
-	public function register_block_script( $script_name, $handle = '', $dependencies = [] ) {
```

Comment:
Note; I removed these as they have been deprecated for several versions and are not being used internally.